tests/test_sensor.py

```
@@ -0,0 +1,93 @@
+    ],
+)
+def test_friendly_usage_time_available(data, expected):
+    coordinator = DummyCoordinator(data)
+    sensor = MyAirFriendlyUsageTime(coordinator)
+    with patch.object(MyAirFriendlyUsageTime, "async_write_ha_state", return_value=None):
+        sensor._handle_coordinator_update()
+    assert sensor.available == expected
+
+
+@pytest.mark.parametrize(
+    "data,sensor_key,expected",
```

Ruff (0.12.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)